docs/get-started-cloud.md

```diff
@@ -9,176 +9,252 @@ weight=2
 
-Let's take a look at how to do this on Digital Ocean.
+As an example, let's take a look at how to create a Dockerized <a href="https://digitalocean.com" target="_blank">Digital Ocean</a> _Droplet_ (cloud server).
 
-## Digital Ocean example
+### Step 1. Create a Digital Ocean account and log in
+
+If you have not done so already, go to <a href="https://digitalocean.com" target="_blank">Digital Ocean</a>, create an account, and log in.
+
+### Step 2. Generate a personal access token
 
 To generate your access token:
 
-1.  Go to the Digital Ocean administrator console and click on "API" in the header.
-2.  Click on "Generate New Token".
-3.  Give the token a clever name (e.g. "machine"), make sure the "Write" checkbox
-    is checked, and click on "Generate Token".
-4.  Grab the big long hex string that is generated (this is your token) and store
-    it somewhere safe.
-
-Now, run `docker-machine create` with the `digitalocean` driver and pass your key to
-the `--digitalocean-access-token` flag.
-
-Example:
-
-    $ docker-machine create \
-        --driver digitalocean \
-        --digitalocean-access-token 0ab77166d407f479c6701652cee3a46830fef88b8199722b87821621736ab2d4 \
-        staging
-    Creating SSH key...
-    Creating Digital Ocean droplet...
-    To see how to connect Docker to this machine, run: docker-machine env staging
-
-For convenience, `docker-machine` will use sensible defaults for choosing
-settings such as the image that the VPS is based on, but they can also be
-overridden using their respective flags (e.g. `--digitalocean-image`). This is
-useful if, for instance, you want to create a nice large instance with a lot of
-memory and CPUs (by default `docker-machine` creates a small VPS). For a full
-list of the flags/settings available and their defaults, see the output of
-`docker-machine create -h`.
-
-When the creation of a host is initiated, a unique SSH key for accessing the
-host (initially for provisioning, then directly later if the user runs the
-`docker-machine ssh` command) will be created automatically and stored in the
-client's directory in `~/.docker/machines`. After the creation of the SSH key,
-Docker will be installed on the remote machine and the daemon will be configured
-to accept remote connections over TCP using TLS for authentication. Once this
-is finished, the host is ready for connection.
-
-To prepare the Docker client to send commands to the remote server we have
-created, we can use the subshell method again:
-
-    $ eval "$(docker-machine env staging)"
-
-From this point, the remote host behaves much like the local host we created in the last section. If we look at `docker-machine ls`, we'll see it is now the
-"active" host, indicated by an asterisk (`*`) in that column:
+  1. Go to the Digital Ocean administrator console and click **API** in the header.
```

Comment:
It feels weird that we provide such detailed explanations about how to setup an account on a given cloud provider.

First, it gives the feeling that we are endorsing its use over the others. Second, this is very basic operations, that should be/are documented by the cloud provider himself.

Could we instead point to their documentation? (that would also guarantee it's up to date instead of adding extra maintenance on us).

Let me know what you think.